### PR TITLE
DBChecker: reconcile the election timeouts when specified

### PR DESCRIPTION
**Pull request by fedepaol**
**- What this PR does and why is it needed**

We may want to update the ovn election timers, and in order to do that
we need to act asynchronously because the election must be completed and
the change must be performed on the leader node. So, we add two optional
parameters for the desired election timer and we change that as part of
the db checker timer.

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: go-controller/pkg/config/config.go [hunk 1]
```diff
@@ -314,8 +314,8 @@ type OvnAuthConfig struct {
 	CACert         string `gcfg:"client-cacert"`
 	CertCommonName string `gcfg:"cert-common-name"`
 	Scheme         OvnDBScheme
-
-	northbound bool
+	ElectionTimer  string `gcfg:"election-timer"`
```
Comment (dcbw) [inline go-controller/pkg/config/config.go:317]: @fedepaol shouldn't this be a uint seconds value, instead of a string?

Comment (fedepaol) [reply go-controller/pkg/config/config.go:317]: Good point, will change it

Comment (fedepaol) [reply go-controller/pkg/config/config.go]: Done

**Pull request merged**